lib/notifications/client/helper_methods.rb

```diff
@@ -0,0 +1,13 @@
+require "base64"
+
+module Notifications
+  class Client
+    module HelperMethods
+      def prepare_upload(file)
```

Comment:
I'd probably put this directly in `Notifications` instead of nesting it within a `Client` class:
```ruby
module Notifications
  def self.prepare_upload(file):
    ...
  end
end
```

And then we can skip the `include` in documentation examples and replace them with `Notifications.prepare_upload(...)`.

Reply:
👍 Updated